### PR TITLE
Readme: link to Docker Desktop and on stopping the server

### PR DESCRIPTION
**Pull request by kkaprolat**
I added a link to Docker Desktop and the command for stopping the server.

Should we also add more elaborate steps on using Docker Desktop? I've never used it, but I imagine you have to point it to the cloned repository and open a shell?

### CODE REVIEW

**Comment by TheNoim** [comment]
> Should we also add more elaborate steps on using Docker Desktop? I've never used it, but I imagine you have to point it to the cloned repository and open a shell?

Sure. But, someone needs to do this all completely crossplatform, because also Docker Desktop differs between Windows and Mac.

**Comment by kkaprolat** [comment]
Really? Oh dear. Well I don't have a Mac, but I could write something tomorrow. If the Windows part works, this would already help quite a few people.

**Comment by TheNoim** [comment]
> Really? Oh dear. Well I don't have a Mac, but I could write something tomorrow. If the Windows part works, this would already help quite a few people.

Do the windows, and I do the rest.

File: Docker.md [hunk 1]
```diff
@@ -30,4 +31,4 @@ services:
       - donotstart
 ```
 
-3. Now run `docker compose up -d --build` or `docker-compose up -d --build`
\ No newline at end of file
+3. Now run `docker compose up -d --build`.
```
Comment (TheNoim) [inline Docker.md:34]: I think it is dangerous to put a `.` after a command. Users tend to copy them.

File: Docker.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## What you need
 
-- Docker (Docker Desktop or on Linux normal Docker)
+- Docker ([Docker Desktop](https://hub.docker.com/editions/community/docker-ce-desktop-windows/) or on Linux normal Docker)
```
Comment (TheNoim) [inline Docker.md]: Extend with https://hub.docker.com/editions/community/docker-ce-desktop-mac/

**Review by TheNoim — CHANGES_REQUESTED** [review]

**Comment by kkaprolat** [comment]
I pushed a first draft for Windows install instructions. I also have images for each step, that I'll include shortly.
Also I noticed that submodules are not downloaded when downloading a ZIP of the repository, this may be a usability issue.

**Comment by TheNoim** [comment]
Sorry, I just want you to know, I am currently ill. I will look into it when I feel good again.

**Comment by kkaprolat** [comment]
Oh, no rush, this can wait. Get well soon!

**Review by TheNoim — APPROVED** [review]
Really like it.

**Pull request merged**